published/011-2018-sep-26.md

```diff
@@ -0,0 +1,62 @@
+## Welcome back to the IPFS Weekly. 👋
+
+The [InterPlanetary File System (IPFS)](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open. Since that’s a pretty large scope, we track development across the ecosystem in this weekly dispatch.
```

Comment:
What does 'addressed by...identities' mean? I know we talk about IPFS being content-addressed, but 'identity-addressed' is a new one for me.

Reply:
"Identity" is sort of a placeholder. We don't have a proper decentralized identity system yet, although several are in the works.

In the meantime, you could think of a domain name as an identity. With IPNS we associate that identifier with a public key (the peerID) which together form a "domain based identity" that IPFS can ask for the current root node of the site.

We may want to change the language here to be "identity" instead of "identities." While identities is grammatically correct, the reference is to the field of "Identity" and the connection isn't as clear when using the plural form.